src/mlnet/strings.Designer.cs

```diff
@@ -10,48 +10,35 @@
 
 namespace Microsoft.ML.CLI {
     using System;
+    using System.Reflection;
     
     
-    /// <summary>
-    ///   A strongly-typed resource class, for looking up localized strings, etc.
-    /// </summary>
-    // This class was auto-generated by the StronglyTypedResourceBuilder
-    // class via a tool like ResGen or Visual Studio.
```

Comment:
weird about the differences in this file. if it looks like nothing of substance changed here, should we revert this file?

Reply:
Agreed that it's odd. There are intended changes, the misspellings. But it seems that rebuilding also re-auto-generated the file, and now the form is a bit different.

Reply:
@Dmitry-A -- any idea why this auto-generated file changed so much?

Reply:
Makes sense that intended changes caused a rebuild. I was noticing that (I think) the misspellings didn't directly change in this file? I was wondering if we reverted the file if the spelling bugs would still be fixed lol. I wonder if this is a Mac vs Windows Visual Studio difference? When I sync your branch on my Windows machine and rebuild the file, it looks like the original file

Reply:
ah I see, the typos were in the comments in this class. perhaps I can push the version I have built on my machine